clog/handlers.py

```diff
@@ -88,6 +89,38 @@ def emit(self, record):
             self.handleError(record)
 
 
+class MonkHandler(logging.Handler):
+    """Handler for sending python standard logging messages to a monk
+        stream.
+
+        .. code-block:: python
+
+            import clog.handlers, logging
+            log = logging.getLogger(name)
+            log.addHandler(clog.handlers.MonkHandler('client_id', 'localhost', 3600, 'stream'))
+
+
+        :param client_id: client id to identify the user logging
+        :param stream: name of the monk stream logs will be sent to
+        :param host: hostname of monk server
+        :param port: port number of monk server
+        """
+
+    def __init__(self, client_id, stream, host=None, port=None):
+        logging.Handler.__init__(self)
+        self.stream = '_clog.{0}'.format(stream)
```

Comment:
I noticed that the MonkLogger can be used directly, without using the MonkHandler, and in that case the prefix is missing. I think we should move it to the MonkLogger instead, which will always be used.